### ISSUE TITLE
docker: flowable-admin env variables for SERVER-CONFIG_... does not rewrited on admin-ui configuration page after changes

### ISSUE THREAD

**Issue by ikusmart** [flowable/flowable-engine#2045]
### Describe the bug
Hello.
When I run the flowable-admin by docker-compose file for the first time, I initialize the variables for SERVER_CONFIG with the following values:
```
...  #describe flowable-rest api container
flowable:
    container_name: flowable
    environment:
     ... #describe flowable rest envs
    ports:
      - port1:8080
    
... #describe flowable-admin container
  flowable-admin-app:
    image: flowable/flowable-admin
    environment:
       ... #other envs for flowable-admin 
        - FLOWABLE_ADMIN_APP_SERVER-CONFIG_PROCESS_SERVER-ADDRESS=http://process-address-old-host
        - FLOWABLE_ADMIN_APP_SERVER-CONFIG_PROCESS_PORT=port1

```
And everything works fine.

But if I try to change the values of these env variables in the docker-compos file (for example from http://process-address-old-host to http://process-address-new-host or from _port1_ to _port2_), after restarting the containers in the admin interface the addresses remain unchanged (by `docker-compose up` or `docker-compose up --build` commands)

Also i don't understand, how can i set login and password for rest-api in flowable-admin-app section in docker-compose (for login to http://process-address-new-host)

### Expected behavior
After restart docker-compose new variables will appear in the flowable-admin

**Comment by yvoswillens** [comment]
Hi Ikusmart,

The ```FLOWABLE_ADMIN_*_SERVER-CONFIG_*``` environment variables are (only) used to override the default application properties when bootstrapping. (In other words when during bootup no endpoint configuration data is present in the database).

What you are describing would be an (additional) set over runtime override env vars. This would be a new feature / PR.

Regarding the username / password question; this can be also be set for bootstrapping. F.e. 
```
FLOWABLE_ADMIN_APP_SERVER-CONFIG_PROCESS_USER-NAME=flowable
FLOWABLE_ADMIN_APP_SERVER-CONFIG_PROCESS_PASSWORD=flowable
```

You can find the default values here;
https://github.com/flowable/flowable-engine/blob/master/modules/flowable-ui-admin/flowable-ui-admin-app/src/main/resources/flowable-default.properties

(In the future please use the forum for asking questions)

Regards,

Yvo

**Comment by wasadigi** [comment]
I also fell into this trap and did not realize that the values were stored in the DB. But there is something else I don't quite understand: the dashes in the environment variable names should cause a problem (it's a forbidden character). I really don't understand how this can work...

**Comment by wasadigi** [comment]
Ok, I learned something about Docker. The Dockerfile is using ENTRYPOINT with the exec form (which is the most common thing to do). What I did not know is that in that case, the environment variables are passed directly to the process and that in that case, dashes are allowed in their names.

However, I think that using dashes in environment variables is misleading and makes it hard to debug things (e.g. when I connect to the running container and look at environment variables, I will not see the values).
